### ISSUE TITLE
[v3] Implement new Concept Exercise: expressions

### ISSUE THREAD

**Issue by ErikSchierboom** [exercism/fsharp#838]
This issue describes how to implement the `expressions` concept exercise for the F# track.

## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Goal

The goal of this exercise is to teach the student the basics of the Concept of Expressions in F#.

## Learning objectives

- Know what expressions are.
- Know how expressions are different from statements.
- Know what pure functions are.
- Know what the `unit` type is.
- Know about the `ignore` function.

## Out of scope

- Monadic expressions.
- Computation expressions.

## Concepts

The Concepts this exercise unlocks are:

- `expressions`: know what expressions are; know how expressions are different from statements; know what pure functions are.
- `unit`: know what the `unit` type is; know about the `ignore` function.

## Prerequisites

This exercise's prerequisites Concepts are:

- `pattern-matching`: know what a `match` expression is.
- `conditionals`: know what an `if/elif/else` expression is.

Any data types used in this exercise (e.g. `strings`) should also be added as prerequisites.

### Hints

- [Expressions vs statements][expressions-vs-statements]: explains the differences between expressions and statements.
- [`unit` type][unit-type]: describes the `unit` type.
- [`ignore` function][ignore-function]: shows the use of the `ignore` function.

### After

- [Expressions vs statements][expressions-vs-statements]: explains the differences between expressions and statements.
- [`unit` type][unit-type]: describes the `unit` type.
- [`ignore` function][ignore-function]: shows the use of the `ignore` function.

## Representer

This exercise does not require any specific representation logic to be added to the [representer][representer].

## Analyzer

This exercise does not require any specific logic to be added to the [analyzer][analyzer].

## Implementing

To implement this exercise, please [follow these instructions](https://github.com/exercism/v3/blob/master/languages/fsharp/reference/implementing-a-concept-exercise.md).

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.

[how-to-implement-a-concept-exercise]: https://github.com/exercism/v3/blob/master/docs/maintainers/generic-how-to-implement-a-concept-exercise.md
[implemented-exercises]: https://github.com/exercism/v3/tree/master/languages/fsharp/exercises/concept/README.md#implemented-exercises
[reference]: https://github.com/exercism/v3/blob/master/reference/concepts/type_inference.md
[reference-example]: https://github.com/exercism/v3/blob/master/reference/types/string.md#implementations
[analyzer]: https://github.com/exercism/fsharp-analyzer
[representer]: https://github.com/exercism/fsharp-representer
[meta-design]: https://github.com/exercism/v3/blob/master/languages/fsharp/exercises/concept/numbers/.meta/design.md
[meta-config.json]: https://github.com/exercism/v3/blob/master/languages/fsharp/exercises/concept/numbers/.meta/config.json
[docs-concept-exercises]: https://github.com/exercism/v3/blob/master/docs/concept-exercises.md
[expressions-vs-statements]: https://fsharpforfunandprofit.com/posts/expressions-vs-statements/
[unit-type]: https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/unit-type
[ignore-function]: https://dotnetcodr.com/2017/06/24/ignoring-the-return-value-of-a-function-in-f/
